examples/docs_beta_snippets/docs_beta_snippets/guides/components/shell-script-component/resolving-resolvable-field.py

```diff
@@ -3,12 +3,12 @@
 from dagster_components import (
     AssetSpecSchema,
     Component,
-    ComponentSchema,
+    ResolvableSchema,
     registered_component_type,
 )
 
 
-class ShellCommandParams(ComponentSchema):
+class ShellCommandParams(ResolvableSchema):
```

Comment:
rename all *Params classes to *Schema in a follow up please

Reply:
added to stack